assets/workflowTemplate.yml

```
@@ -1,23 +1,31 @@
-on: workflow_dispatch
-
 name: test
 
+on: workflow_dispatch
+
 env:
   FOUNDRY_PROFILE: ci
-
+  
 jobs:
   check:
+    strategy:
+      fail-fast: false
```

We should fail fast - doesn't make sense not to. If the build fails, tests would also fail

---

this is for the workflow jobs, if one of the jobs in the matrix expansion fails, the rest of the jobs will be cancelled. Turning off fail-fast helps to more easily iterate on the workflow action.

Also its irrelevant actually as this workflow is pretty short and there are no other additional steps, i'll remove it.

---

> We should fail fast - doesn't make sense not to. If the build fails, tests would also fail

default is now `true`